lib/oelite/runq.py

```diff
@@ -557,6 +557,11 @@ def get_runabletask(self):
             return None
         return self.cookbook.get_task(id=task_id)
 
+    def get_runabletasks(self):
+        self.update_runabletasks()
+        ret = [self.cookbook.get_task(id=task_id) for task_id in self.runable]
+        self.runable = []
```

Comment:
... after refactoring the update from sql part into .update_runabletasks, which is where you'll find self.runable being populated